### PR TITLE
Tidy Fig 6

### PR DESCRIPTION
**Pull request by hyanwong**
Put nodes in the same order in all figures, remove whitespace.

### CODE REVIEW

File: illustrations.py [hunk 1]
```diff
@@ -844,5 +854,19 @@ def get_edge_colors(ts):
 cli.add_command(simplification)
 cli.add_command(inference)
 
+
+### JSON data below from running tskit_arg_visualizer on the tree sequences above
+
+tskit_arg_visualizer_json = dict(
```
Comment (jeromekelleher) [inline illustrations.py:860]: This is pretty nasty - maybe we could save these as individual (formatted) JSON files? Massive single line JSON dumps are not very nice for git.

**Review by jeromekelleher — COMMENTED** [review]
Looks great - lets save the JSONs out of the Python code though. Piping through jq should give reasonable formatting.

File: illustrations.py (hunk 1 above)
Comment (hyanwong) [reply illustrations.py:860]: Yeah - happy to save them as (separate) plain text files. I'm not keen to format the JSON though. I just want to copy and paste from within the `tskit_arg_visualiser`. If we change them, chance are the entire JSON will change anyway, so I don't think we should worry about git diffs (and they might seem massive visually, but they are not many bytes to store)

Comment (hyanwong) [reply illustrations.py:860]: I think of it as git's fault if it doesn't like strings with no line breaks 😄

Comment (hyanwong) [reply illustrations.py]: OK, now as separate json files.

**Comment by hyanwong** [comment]
I was in the middle of changing the `simplification` plot to address #337, so there's also a small change rolled in to do with that (I didn't want to go back and undo it all).

**Pull request merged**